ui-v2/tests/helpers/type-to-url.js

```diff
@@ -11,6 +11,9 @@ export default function(type) {
     case 'proxy':
       requests = ['/v1/catalog/connect'];
       break;
+    case 'intention':
+      requests = ['/v1/connect/intentions'];
+      break;
```

Comment:
This enables us to mock intentions out from within our feature tests themselves. I'm actually surprised we've got this far just relying on the default mocks!